src/settings/utils/grouping.js

```diff
@@ -150,3 +150,21 @@ export const groupById = (arr) => {
     return acc;
   }, {});
 };
+
+
+/**
+ * Get object by findind intersection between application capabilities and intersecting list
```

Comment:
typo: `s/findind/finding/`

Reply:
Maybe include a description of the data transform to help clarify that we are transforming a list into an object with corresponding properties? e.g. given a list of capabilities like
```
[{id: foo, {id: bar}, {id: baz}]
```
and an intersecting list like `[foo, bar, zoink, zeta]`, return an object like
```
{ foo: true, bar: true }
```